tests/unit/src/test/scala/tests/ToplevelSuite.scala

```diff
@@ -24,13 +26,14 @@ abstract class ToplevelSuite(
         val ls = FileIO.listAllFilesRecursively(dir)
         ls.files.foreach { relpath =>
           val reluri = relpath.toURI(isDirectory = false).toString
-          Mtags.topLevelSymbols(dir.resolve(relpath), dialect).foreach {
-            toplevel =>
-              // do not check symtab for Scala 3 since it's not possible currently
-              if (symtab.info(toplevel).isEmpty && dialect != Scala3) {
-                missingSymbols += toplevel
-              }
-              toplevels += s"$reluri -> $toplevel"
+          val path = dir.resolve(relpath)
+          val fileSymtab = symtab(path)
+          Mtags.topLevelSymbols(path, dialect).foreach { toplevel =>
+            // do not check symtab for Scala 3 since it's not possible currently
```

Comment:
Doesn't this change mean that you can check it for Scala 3 too now?

Reply:
ach, I think it should